gammapy/spectrum/flux_point.py

```diff
@@ -626,10 +625,12 @@ class FluxPointEstimator(object):
     """
 
     def __init__(self, obs, groups, model):
-        self.obs = obs
+        if isinstance(obs, SpectrumObservation):
+            obs = SpectrumObservationList([obs])
+        self._obs = SpectrumObservationList(obs)
```

Comment:
There was a reason to introduce the `FluxPointEstimator.obs` setter, probably there were callers that assigned to it and needed it. But OK, all tests pass now, and generally I like that it's simpler and agree it shouldn't be needed.
So merging this now.